testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6618,6 +6618,60 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkNull("pow(cast(null as integer), 2)");
   }
 
+  @Test void testInfinity() {
+    final SqlOperatorFixture f = fixture();
+    f.checkScalar("CAST('Infinity' AS DOUBLE)", "Infinity",
```

Comment:
Would it be better to align the case of keywords, such as keep lower case as other test cases?

Reply:
Done, let me know if you have other suggestions!